WebKitBrowser/CMakeLists.txt

```diff
@@ -15,33 +15,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.3)
-find_package(WPEFramework)
-
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/../cmake/")
-
-include(CmakeHelperFunctions)
-
-# Library installation section
-string(TOLOWER ${NAMESPACE} STORAGE_DIRECTORY)
-
-if(BUILD_REFERENCE)
-    add_definitions(-DBUILD_REFERENCE=${BUILD_REFERENCE})
-endif()
```

Comment:
This is needed to make it possible to build just a browser plugin. Please don't remove it.

Reply:
This needs to be in by definition, would expect it not even to compile with out this. Will discuss with @svdmaar if it is done diffrently now.